source/index.html.erb

```diff
@@ -106,12 +106,22 @@ title: OpenShift Origin - Open Source Container Application Platform
   </div>
   <div class="">
     <div class="container wow fadeInUp">
-      <div class="row text-center">
-        <div class="col-xs-2 col-sm-2 col-md-2 col-lg-2"></div>
-        <div class="col-xs-8 col-sm-8 col-md-8 col-lg-8">
-          <pre class="code">$ oc cluster up    <a id="get-started" href="https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md" target="_blank">Get Started</a></pre>
+      <div class="row">
```

Comment:
This was refactored from using placeholder columns to using a centered container column